Donation-platform/src/main/webapp/trendingScript.js

```diff
@@ -32,11 +34,12 @@ function queryServer() {
       })
       .then((charities) => {
         // Convert the charity details from a json representation to strings.
-        // For simplicity, this method will (for now) only output the charity names to update the display.
+        // Currently extracts the name and the corresponding tags of the charities for display.
         // TODO: update this method to extract all necessary details of the charity to update the display.
         const out = [];
         charities.forEach((charity) => {
-          out.push(charity.name);
+          out.push([charity.name, charity.tags]);
```

Comment:
why not just push the whole charity object instead of splitting and keeping the 2-element array?

Reply:
yeah, that would be better. fixed and re-committed.